lib/letter_opener/configuration.rb

```diff
@@ -3,7 +3,7 @@ class Configuration
     attr_accessor :location, :message_template
 
     def initialize
-      @location = Rails.root.join('tmp', 'letter_opener') if defined?(Rails)
+      @location = Rails.root.join('tmp', 'letter_opener') if defined?(Rails) && Rails.respond_to?(:root) && Rails.root
```

Comment:
can you please remove `&& Rails.root`? I think it's extra check we don't really need. `respond_to?` should be enough